tests/javascript/newsletter_editor/blocks/social.spec.js

```diff
@@ -6,7 +6,7 @@ define([
   'newsletter_editor/blocks/social',
   'backbone'
 ], function (EditorApplication, SocialBlock, Backbone) {
-  describe('Social', function () {
+  describe.only('Social', function () {
```

Comment:
Remove `.only` from here. It is very easy to forget. I wish we could add an eslint rule for that.